src/envo/envo-edit.owl

```diff
@@ -11221,13 +11221,17 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_00000485> ObjectSomeValuesFrom(<
 
 # Class: <http://purl.obolibrary.org/obo/ENVO_00000486> (shoreline)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "USGS:SDTS"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00000486> "The line of contact between a body of water and the land."^^xsd:string)
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://www.gesamp.org/publications/guidelines-for-the-monitoring-and-assessment-of-plastic-litter-in-the-ocean>) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "USGS:SDTS"^^xsd:string) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_00000486> "A compound astronomical body part which lies between a large body of water and the land."^^xsd:string)
```

Comment:
@meichen-liu you've added GESAMP to the definition xrefs - what was their definition  or description of shoreline? 

We have be clear on how [shoreline](http://purl.obolibrary.org/obo/ENVO_00000486) relates to [shore](http://purl.obolibrary.org/obo/ENVO_00000304). 

Most definitions agree that the shoreline is where the land and water "meet", although there is no way to threshold this. This makes it a fiat boundary. We could think of this as a [two-dimensional continuant fiat boundary](http://purl.obolibrary.org/obo/BFO_0000146). 

This will let use it in [has 2D boundary](http://purl.obolibrary.org/obo/RO_0002002) and [2D boundary of](http://purl.obolibrary.org/obo/RO_0002000) relations. 

So, let's try: 

> A two-dimensional continuant fiat boundary which is located between a landmass and a water body.

Please add a comment noting that 1) this term is usually used for larger water bodies like lakes and oceans, 2) that the actual spatial extent of a shoreline, and the sharpness of its boundaries, is often arbitrarily or operationally defined, and 3) this term is for the physical shoreline, not the one- or two-dimensional representation of shorelines.

Reply:
@pbuttigieg the GESAMP report describe `shoreline` as

> The shoreline is the interface between land and sea.

I will first edit the definition accordingly to your suggestion!